crashlogger.js

```diff
@@ -23,9 +23,9 @@ let hadException = false;
  * Logs when a crash happens to console, then e-mails those who are configured
  * to receive them.
  *
- * @param {Error} err
+ * @param {NodeJS.ErrnoException} err
```

Comment:
I mean, we pass straight `Error`s to Crashlogger, which would now be problematic?